src/views/Login/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import Layout from "../../components/Layout";
+import "./Login.css";
+import * as tokenService from "../../scripts/token.js";
+
+export default class Login extends Component {
+  state = {
+    username: "",
+    password: ""
+  };
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  handleLogin = async event => {
+    event.preventDefault();
+    // await fetch("https://localhost:5001/api/token", {
```

Comment:
assuming this is just on standby for when we start establishing server connection, then it's pressence is fine 👍

Reply:
Remove it 😄